src/core_atmosphere/physics/mpas_atmphys_driver_convection.F

```diff
@@ -652,19 +672,29 @@ subroutine convection_from_MPAS(dt_dyn,configs,mesh,sfc_input,diag_physics,tend_
  call mpas_pool_get_array_gpu(tend_physics,'rqvcuten',rqvcuten)
  call mpas_pool_get_array_gpu(tend_physics,'rqccuten',rqccuten)
  call mpas_pool_get_array_gpu(tend_physics,'rqicuten',rqicuten)
-!$acc update host(cuprec, raincv, rthcuten, rqvcuten, rqccuten, rqicuten)
 
  do j = jts,jte
```

Comment:
This will launch jte number of kernels. @Pranay-Reddy-Kommera @cponder Should we move the parallel region outside?